eng/Versions.props

```diff
@@ -16,6 +16,8 @@
     <DotNetFinalVersionKind Condition="'$(StabilizePackageVersion)' == 'true'">release</DotNetFinalVersionKind>
     <!-- Opt-in/out repo features -->
     <UsingToolMicrosoftNetCompilers>true</UsingToolMicrosoftNetCompilers>
+    <!-- TODO: Upgrade compiler version to enable static lambdas; remove this once the employed SDK uses a new enough version. -->
+    <MicrosoftNetCompilersToolsetVersion>3.8.0-1.20360.1</MicrosoftNetCompilersToolsetVersion>
```

Comment:
@safern, is this the right way to do this?

Reply:
Yeah.It seems like arcade just updated an hour ago so we should remove this whenever we consume a new Arcade.

https://github.com/dotnet/arcade/blob/master/eng/Versions.props#L34

Would you mind updating the comment?